app/src/main/res/layout/sign_in_fragment.xml

```diff
@@ -0,0 +1,126 @@
+
+            <com.google.android.material.textfield.TextInputEditText
+                android:id="@+id/et_login"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:hint="@string/text_hint_login"
+                android:inputType="textEmailAddress"
+                android:paddingStart="@dimen/normal_100"
+                android:singleLine="true"
+                android:text="@={viewModel.loginEditTextData}"
+                android:textColor="@color/black"
+                android:textColorHint="@color/color_hint"
+                android:theme="@style/MyEditText" />
+
+        </com.google.android.material.textfield.TextInputLayout>
+
+
+        <com.google.android.material.textfield.TextInputLayout
+            android:id="@+id/til_password"
+            android:layout_width="0dp"
+            android:layout_height="@dimen/large_150"
+            android:layout_marginTop="@dimen/normal_150"
+            android:background="@drawable/item_background"
+            android:hint="@string/text_hint_password"
+            app:hintTextAppearance="@style/MyHintText"
+            app:layout_constraintEnd_toEndOf="@id/til_login"
+            app:layout_constraintStart_toStartOf="@id/til_login"
+            app:layout_constraintTop_toBottomOf="@id/til_login">
+
+            <com.google.android.material.textfield.TextInputEditText
+                android:id="@+id/et_password"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:hint="@string/text_hint_password"
+                android:inputType="textPassword"
+                android:paddingStart="@dimen/normal_100"
+                android:singleLine="true"
+                android:text="@={viewModel.passwordEditTextData}"
+                android:textColor="@color/black"
+                android:textColorHint="@color/color_hint"
+                android:theme="@style/MyEditText" />
+        </com.google.android.material.textfield.TextInputLayout>
+
+
+        <Button
+            android:id="@+id/btn_login"
+            android:layout_width="0dp"
+            android:layout_height="@dimen/large_150"
+            android:layout_marginTop="@dimen/large_350"
+            android:layout_marginBottom="@dimen/large_175"
+            android:background="@drawable/btn_login"
+            android:fontFamily="@font/montserrat_light"
+            android:onClick="@{()-> viewModel.login()}"
+            android:text="@string/btn_login"
+            android:textSize="@dimen/normal_text_size"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="@id/til_password"
+            app:layout_constraintStart_toStartOf="@id/til_password"
+            app:layout_constraintTop_toBottomOf="@id/til_password" />
+
+    </androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
Not sure that this layout fits to smaller/bigger screen size, it'd be better to use constraint weight/chains/barriers for it